src/main/java/uk/gov/companieshouse/model/dto/DissolutionGetResponse.java

```diff
@@ -0,0 +1,130 @@
+package uk.gov.companieshouse.model.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.model.db.DissolutionGetDirector;
+import uk.gov.companieshouse.model.enums.DissolutionStatus;
+import uk.gov.companieshouse.model.enums.DissolutionType;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+public class DissolutionGetResponse {
+
+    private String ETag;
+
+    private String kind;
+
+    private DissolutionLinks links;
+
+    @JsonProperty("application_status")
+    private DissolutionStatus dissolutionStatus;
+
+    @JsonProperty("application_reference")
+    private String applicationReference;
+
+    @JsonProperty("application_type")
+    private DissolutionType dissolutionType;
```

Comment:
Ahhh now I see why you chosen ApplicationStatus and ApplicationType. I think you were right to change these because they **do** make sense from an API swagger perspective. Sorry, I shouldn't have asked you to change this. The annotations confused me when they were bunched together.

You can change this back if you wish, or leave it as DissolutionStatus/Type/ I'll leave the choice up to you.